rustup-toolchain

```diff
@@ -42,7 +42,7 @@ fi
 
 # Install and setup new toolchain.
 rustup toolchain uninstall miri
-rustup-toolchain-install-master -n miri -c cargo -c rust-src -c rustc-dev -c llvm-tools "$@" -- "$NEW_COMMIT"
+rustup-toolchain-install-master -n miri -c cargo -c rust-src -c rustc-dev -c llvm-tools -c rustfmt "$@" -- "$NEW_COMMIT"
```

Comment:
This will increase the dowload size, which I would rather avoid. I think we should use the nightly rustfmt.

Reply:
I keep running this command like that every time ^^ I never bump nightly, so the rustfmt diffs can diverge

Reply:
To be clear, we are talking about a 1.79% increase in bytes downloaded. I think that is *obviously* worth it for the contributor experience of not clashing on formatting depending on what nightly somebody has installed.

Reply:
Right. Yea, sticking with nightly works well enough to just keep doing it

Reply:
> That takes more than 10x of the current runtime of the fmt job.

I pushed a version that is twice as fast as the current `fmt` job while still using the toolchain selected in `rust-toolchain`.

## Before:

![Screenshot from 2022-06-22 10-49-52](https://user-images.githubusercontent.com/1940490/175103746-72d6b115-5500-4f11-977f-50d60a5fa5ba.png)

## After:

![Screenshot from 2022-06-22 10-50-21](https://user-images.githubusercontent.com/1940490/175103763-ac682896-6ce6-46fb-aab4-2d60fb2fed2d.png)

Reply:
This still installs rustfmt from the commit